pkg/distro/test_distro/distro.go

```diff
@@ -392,47 +389,3 @@ func NewRegistry() *distroregistry.Registry {
 	registry.SetHostArchName(TestArchName)
 	return registry
 }
-
-// New2 returns new instance of TestDistro named "test-distro-2".
-func New2() *TestDistro {
```

Comment:
This is needed by osbuild_composer tests in `client/unit_test.go` and `weldr/api_test.go`

Reply:
I was doing the same just now and I think it might be better to copy `test-distro-2` back into osbuild-composer.